types/dialogflow/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "quotemark": [true, "double", "avoid-escape"]
```

Comment:
Added double quote enforcement via tslint with the `avoid-escape` argument so that it's ok to use a single quote in a string that would cause heavy escaping, such as the HTML case.

Reply:
Please don't add exceptions to the lint rules (especially since it doesn't seem to be used here). Instead, consider a PR to dtslint.

Reply:
Ok no prob. Just pushed a commit for removal.